ui_framework/components/table/table_header_check_box_cell.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTableHeaderCell } from './table_header_cell';
+
+export const KuiTableHeaderCheckBoxCell = ({ onChange, isChecked, className, ...rest }) => {
```

Comment:
Can we add a test file for this component, to verify that custom classes and other attributes can be supplied?